tests/functional/job-file-trap/03-user-trap.t

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) 2008-2019 NIWA & British Crown (Met Office) & Contributors.
```

Comment:
You may want to update the copyright date here, it's been a while since I've wrote this 😉.